tests/conftest.py

```diff
@@ -38,6 +39,7 @@ def run_pipeline(
     if 'NEPTUNE_CUSTOM_RUN_ID' in os.environ:
         del os.environ['NEPTUNE_CUSTOM_RUN_ID']
 
+    configure_project(project)
```

Comment:
Is it necessary?

Reply:
It stores the package name in settings and fixes the `No module named 'None'` error.

Also, it's used in https://github.com/neptune-ai/kedro-neptune/blob/main/examples/planets/src/planets/__main__.py#L39-L39 so it can't be THAT wrong ;)